src/client/ui/styles.less

```diff
@@ -659,6 +670,55 @@
                 }
             }
         }
+
+        .selector-panel-toggle-button {
+            background: white;
+            position: absolute;
+            height: 21px;
+            width: 70px;
+            padding: 3px 5px;
+            top: 2px;
+            right: 2px;
+            border-radius: 4px;
+            overflow: hidden;
+            z-index: @rootZIndex - 2;
+
+            &:hover {
+                background-color: #cfcfcf;
+            }
+
+            span::after {
+                content: "Hide Picker";
+                cursor: pointer;
+                position: absolute;
+                display: flex;
+                align-items: center;
+                justify-content: center;
+                width: 100%;
+                height: 100%;
+                left: 0;
+                top: 0;
+            }
+
+            &--show {
+                background-color: #283545;
+                color: white;
+                font-weight: bold;
+                top: -24px;
+
+                &:hover {
+                    background-color: #36B6E5;
+                }
+    
+                span::after {
+                    content: "Show Picker";
+                }
+    
+                & ~ .selector-inspector-panel-container {
+                    display: none;
+                }
+            }
+        }
     }
```

Comment:
![image](https://github.com/DevExpress/testcafe/assets/1678902/e36cacd4-cfa7-46f1-b7fa-6cd829016a04)
These two buttons should look the same.

![image](https://github.com/DevExpress/testcafe/assets/1678902/f42ddb8c-1097-424e-b5e8-7067fc78620a)
I believe this stripe should not be rendered.